sdk/core/switcher/entry.S

```diff
@@ -777,7 +777,7 @@ exception_entry_asm:
 	// Restore the stack pointer.  All other spilled values are spilled there.
 	clc                csp, TrustedStackFrame_offset_csp(ct1)
 	// Update the current frame offset.
-	csw                t2, TrustedStack_offset_frameoffset(ctp)
+	csh                t2, TrustedStack_offset_frameoffset(ctp)
```

Comment:
Was this previously a bug where this write was overflowing into the padding?

Reply:
Yes, but it was benign because it stored zeroes into padding, so embarrassing, but not a security issue.